UKHomeOffice/sgmr-service
# SPCR-556 Update pleasure craft form registration hint text

### AC / Description of changes
PC Hint text to be updated


### To Test
- go to add a pleasure craft or edit a pleasure craft
- hint text should show as:

![image](https://user-images.githubusercontent.com/724554/183096125-f4d57aec-77ce-43b6-8bd3-f2e6a0680535.png)


### Notes

---

**Remember to merge to feature branches not `master`.**
